src/meta/client/MetaClient.cpp

```diff
@@ -897,6 +916,107 @@ MetaClient::dropEdgeSchema(GraphSpaceID spaceId, std::string name) {
     }, true);
 }
 
+folly::Future<StatusOr<TagIndexID>>
+MetaClient::createTagIndex(GraphSpaceID spaceID, std::string name,
+                           std::map<std::string, std::vector<std::string>> fields) {
+    cpp2::IndexProperties properties;
+    properties.set_fields(fields);
```

Comment:
if the parameter is passed by value, here should add std::move, and when call createTagIndex, the caller should call std::move too, else it will call copy constructor function.